pkgs/applications/misc/ethereum/cpp/default.nix

```diff
@@ -0,0 +1,134 @@
+{ stdenv
+, fetchgit
+, cmake
+, boost
+, gmp
+, jsoncpp
+, leveldb
+, cryptopp
+, cudatoolkit
+, nvidia_x11
```

Comment:
I'm thinking "nvidia_x11" and "cudatoolkit" should be optional? Since they're unfree, Hydra isn't going to build this derivation unless they're made optional.